tests/cxx/OperatorPythonTest.cxx

```diff
@@ -27,7 +27,7 @@ class OperatorPythonTest : public ::testing::Test
   void SetUp() override
   {
     dataObject = vtkDataObject::New();
-    pythonOperator = new OperatorPython();
+    pythonOperator = new OperatorPython(nullptr);
```

Comment:
I think this might crash when ran because `OperatorPython` tries to access its parent `DataSource` [in its constructor](https://github.com/OpenChemistry/tomviz/blob/e3e8e01fe33ee51ad039f8a906ce11f1d9bbcd49/tomviz/operators/OperatorPython.cxx#L160). :slightly_frowning_face:

Reply:
I think your right

Reply:
I guess we can either make a `DataSource` here, or maybe modify the python operator constructor to check if its parent exists before calling functions on it?

Reply:
yep, I am looking a creating a `DataSource` but it looking like a rabbit hole ...